src/main/java/com/bieganski/ox/checkers/Judge.java

```diff
@@ -0,0 +1,45 @@
+package com.bieganski.ox.checkers;
+
+import com.bieganski.ox.App;
+import com.bieganski.ox.BoardListener;
+import com.bieganski.ox.GameListener;
+import com.bieganski.ox.model.Field;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.TreeSet;
+
+/**
+ * Board checker, Judge initialize win checkers
+ * to check if added symbol is winning move.
+ */
+public class Judge implements BoardListener {
+  private List<WinChecker> checkers = new ArrayList<>();
+  private int boardSize;
+
+  private GameListener gameListener;
+
+  /**
+   * Constructor for Judge class.
```

Comment:
Konstruktor kopiujący nie powinien mieć dokumentacji, chyba, że robi coś nietypowego. Parametry udokumentowane napisaniem swej nazwy raz jeszcze to nie jest "coś nietypowego".

Reply:
Javadoc został usunięty